MeasureAuthoringTool/mat-drupal
# Configure drupal menus

## Description
Creates or configures the necessary menus in Drupal. Adds several block twig files to hook into Pattern Lab. Adds Pathauto module, configured path rule for Basic Page content.

## To Test
1. `composer install` to get the new dependencies
2. `docker-compose exec drupal drush cim -y` to import config changes
3. Compile pattern lab `npm start` from the `html/themes/custom/emat/` directory
4. You'll need to add some menu items to the Main nav, Secondary nav, and Footer nav (the menus are config, but the entries are content, so can't be imported/exported):
  - `/admin/structure/menu/manage/main`
  - `/admin/structure/menu/manage/secondary`
  - `/admin/structure/menu/manage/footer`
5. View the site's homepage, and see that the Drupal menus are themed correctly (something like attached screenshot)

<img width="1678" alt="Screen Shot 2020-02-20 at 8 39 07 AM" src="https://user-images.githubusercontent.com/1919263/74944963-86475400-53bc-11ea-8d27-bb10784de988.png">
